lib/board.rb

```diff
@@ -31,31 +31,31 @@ def valid_coordinate?(coordinate)
 
   def valid_placement?(ship, coordinates)
```

Comment:
@amymoller I refactored the `valid_placement?` method to be cleaner/more readable.

Reply:
Niiiice; that looks really clean!  I'm really glad we went over refactoring.  I practiced it a bit with Graham yesterday too during our session; it's so handy.